src/d3/scatterplot.d3.ts

```diff
@@ -51,7 +52,29 @@ export class Scatterplot{
       return color;
     });
   }
+  private createGridPlane()
+  {
+    const xGrid = document.createElement('a-entity');
```

Comment:
Add test.

Reply:
just went ahead and added tests over from the `addingPlanesVR` branch... I had thought I pulled from there before submitting, but I guess not. Tests are in the changed files now tho!